docs/src/modules/components/demos/data-grid/Inventory/InventoryDashboard.tsx

```diff
@@ -0,0 +1,257 @@
+import React from 'react';
+import {
+  DataGridPremium,
+  GridColDef,
+  GridRenderCellParams,
+  GridEventListener,
+  useGridApiRef,
+  GridAggregationFunction,
+  GRID_AGGREGATION_FUNCTIONS,
+  GRID_AGGREGATION_ROOT_FOOTER_ROW_ID,
+} from '@mui/x-data-grid-premium';
+import { Box, Typography, Rating, Chip, Paper } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { ThemeProvider } from '@mui/material/styles';
+import CheckCircleOutlineIcon from '@mui/icons-material/CheckCircleOutline';
+import CancelOutlinedIcon from '@mui/icons-material/CancelOutlined';
+import HourglassEmptyIcon from '@mui/icons-material/HourglassEmpty';
+import { DemoContainer } from '../DemoContainer';
+import { inventoryTheme } from './theme';
+import { Product, products } from './data/products';
+import { ProductDetailPanel } from './DetailPanel';
+import { InventoryToolbar } from './InventoryToolbar';
+import { inventoryDataGridStyles } from './styles';
+import { CustomExpandIcon, CustomCollapseIcon } from './icons';
+
+const StatusChip = styled(Chip)(({ theme }) => ({
+  borderRadius: '6px',
+  height: '30px',
+  fontWeight: 500,
+  '& .MuiChip-label': {
+    padding: '0 8px 0 10px',
+    fontSize: '0.875rem',
+  },
+  '& .MuiChip-icon': {
+    fontSize: '0.875rem',
+  },
+  '&.in_stock': {
+    color: theme.palette.success.dark,
+    backgroundColor: 'rgba(46, 125, 50, 0.1)',
+    '& .MuiChip-icon': {
+      color: theme.palette.success.main,
+    },
+  },
+  '&.out_of_stock': {
+    color: theme.palette.error.dark,
+    backgroundColor: 'rgba(211, 47, 47, 0.1)',
+    '& .MuiChip-icon': {
+      color: theme.palette.error.main,
+    },
+  },
+  '&.restocking': {
+    color: theme.palette.warning.dark,
+    backgroundColor: 'rgba(255, 193, 7, 0.1)',
+    '& .MuiChip-icon': {
+      color: theme.palette.warning.main,
+    },
+  },
+}));
+
+const ProductImage = styled('img')({
+  width: 60,
+  height: 60,
+  objectFit: 'cover',
+  borderRadius: 4,
+});
+
+const columns: GridColDef<Product>[] = [
+  {
+    field: 'product',
+    groupable: false,
+    aggregable: false,
+    headerName: 'Product',
+    flex: 1,
+    minWidth: 250,
+    renderCell: (params: GridRenderCellParams<Product, string>) =>
+      params.id === GRID_AGGREGATION_ROOT_FOOTER_ROW_ID ? null : (
+        <Box sx={{ display: 'flex', alignItems: 'center', gap: 1 }}>
+          <ProductImage src={params.row.image} alt={params.value} />
+          <Typography sx={{ fontSize: '0.875rem', fontWeight: '500' }}>{params.value}</Typography>
+        </Box>
+      ),
+  },
+  { field: 'sku', groupable: false, aggregable: false, headerName: 'SKU', width: 120, flex: 1 },
+  {
+    field: 'status',
+    groupable: false,
+    aggregable: false,
+    headerName: 'Status',
+    width: 120,
```

Comment:
```suggestion
    minWidth: 150,
```

To make statuses visible